declarations.d.ts

```diff
@@ -1660,6 +1660,14 @@ interface IVersionInformation {
 	 * The latest available version of the component.
 	 */
 	latestVersion: string;
+	/**
+	 * The message that will be displayed.
+	 */
+	message?: string;
+	/**
+	 * The type of the component. Can be UpToDate, UpdateAvailable, NotInstalled.
+	 */
+	type?: any;
```

Comment:
We could use an `enum` here

Reply:
We actually can't use an enum here, but we can specify the type for example 

```ts
type?: "UpToDate"|"UpdateAvailable"|"NotInstalled";
```

and in CLI we can declare:

```ts
export enum VersionInformationType {
	UpToDate = "UpToDate",
	UpdateAvailable = "UpdateAvailable",
	NotInstalled = "NotInstalled"
}
```


This way the F2 refactor functionality will still work and we'd have a bit more security during compilation